core/platform/image/gae_image_services.py

```diff
@@ -79,9 +78,12 @@ def compress_image(image_content, scaling_factor):
                     MAX_RESIZE_DIMENSION_PX, float(max(width, height))))
             new_width = int(width * new_scaling_factor)
             new_height = int(height * new_scaling_factor)
-        return images.resize(
-            image_data=image_content,
-            width=min(new_width, MAX_RESIZE_DIMENSION_PX),
-            height=min(new_height, MAX_RESIZE_DIMENSION_PX))
+        img = img.resize((min(new_width, MAX_RESIZE_DIMENSION_PX),
```

Comment:
Do not redeclare variables. Perhaps call the new thing resized_img or similar.

Also, when breaking across multiple lines, break after '(' -- see the [coding style guide](https://github.com/oppia/oppia/wiki/Coding-style-guide#python) on the wiki. (I think I might have mentioned this before; in any case it's probably worth reading that guide if you haven't already.)

Reply:
Done